src/content/en/fundamentals/design-and-ui/input/touch/_code/states-example.html

```diff
@@ -1,33 +1,21 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
```

Comment:
I don't think those files are used at all. Looking at the demo the CSS is standlone and no MDL is used.